src/main/java/sky/exception/TextNoMeaningException.java

```diff
@@ -0,0 +1,13 @@
+package sky.exception;
+
+public class TextNoMeaningException extends Exception {
```

Comment:
Perhaps add a header comment for this class?

Reply:
Good suggestion!